### DISCUSSION TITLE
Fuzz with Atheris?

### DISCUSSION THREAD

**Discussion by davelab6** [fonttools/fontbakery#3114] [Q&A]
Similarly to #3014 and #3018, there's a new python fuzzer announced today @ https://opensource.googleblog.com/2020/12/announcing-atheris-python-fuzzer.html that we should metatest fontbakery with.

**Comment by felipesanches** [comment]
Atheris knows nothing about fonts. FontBakery always needs font files as inputs. I am not sure how would this tool perform fuzzing if it has no knowledge on how to synthesize fake font files to feed FontBakery.

Or maybe I did not fully understand what Atheris does...
